src/python-common/ceph/deployment/hostspec.py

```diff
@@ -129,6 +129,8 @@ def __str__(self) -> str:
 
     def __eq__(self, other: Any) -> bool:
         # Let's omit `status` for the moment, as it is still the very same host.
+        if not isinstance(other, HostSpec):
+            return NotImplemented
```

Comment:
I think you saw this on another PR as well, so whatever we needed there we might need here (unless that gets merged first maybe? Not sure)
```
ceph/deployment/hostspec.py:135:16: E127 continuation line over-indented for visual indent
ceph/deployment/hostspec.py:136:16: E127 continuation line over-indented for visual indent
ceph/deployment/hostspec.py:137:16: E127 continuation line over-indented for visual indent
```

Reply:
fixed